### ISSUE TITLE
Random disconnect with SSL error

### ISSUE THREAD

**Issue by Tedward747** [philippe44/AirConnect#276]
Using aircast-win on Windows 10 pro to stream from an iPhone to a Google Nest Mini, everything works perfectly until it suddenly doesn't anymore.  Audio just stops, although the iPhone shows that it is still connected & playing.  I've had it running for ~4 hours with no issues, but typically it dies in the range of 10-30 minutes of playback.  This is the error that's being thrown in aircast:

```
[17:14:16.326] read_bytes:181 [s-02977300]: SSL error code 6 (err:0)
[17:14:16.326] CastSocketThread:710 [0234C5B0]: SSL connection closed
```

I've tried using no flags, as well as setting latency to both 500ms and 5000ms with no discernible difference, not sure what else to try.

**Comment by philippe44** [comment]
It means that the periodic SSL renegociation failed. it might mean there is something funny with the SSL library on Win10. Try to either not use my ssleay and libeay that are in the AirConnect bin directory or, if you in fact did not copy them, do the opposite: use them!

**Comment by Tedward747** [comment]
At first I was using the dll's in your provided bin directory.  I sourced the two DLL's from elsewhere and replaced yours and then the exe wouldn't even start, gave an error about not being able to find the libraries.  Tried a few different versions with the same error.

Out of curiosity I spun up a fresh Ubuntu VM, apt upgraded and then downloaded aircast-x86-64 and I'm getting the exact same behaviour that I described in my first post, so I'd assume it's something to do with my network or the speaker.

I then borrowed an android from a friend and managed to successfully cast for hours over my network.
